datadog_checks_dev/datadog_checks/dev/tooling/commands/meta/snmp/generate_traps_db.py

```diff
@@ -294,8 +293,9 @@ def generate_trap_db(compiled_mibs, compiled_mibs_sources, no_descr):
         for trap in traps.values():
             trap_name = trap['name']
             trap_oid = trap['oid']
+            trap_mib_name = file_content['meta']['module']
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** This doesn't need to be defined inside of the `for trap in traps.values()` loop. Can we move it one level up? It feels odd that we recreate this variable on each iteration, especially if we reuse it on line 332 which is one level higher
‏